platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/ConnectionServer.java

```diff
@@ -90,7 +94,10 @@ private void listen(final ServerSocket serverSocket) throws InterruptedException
         while (!serverSocket.isClosed()) {
             try {
                 final Socket clientSocket = serverSocket.accept(); // listen, waiting until someone connects
-                incomingConnPool.submit(() -> newConnectionHandler.accept(clientSocket));
+                incomingConnPool.submit(() -> {
```

Comment:
I think we want to avoid having code inside lambdas where possible

Reply:
wrapped in its own class now.